bin/metadata_validation.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import json
+
+import synapseclient
+from synapseclient import Entity
+
+
+def get_synaspe_file(syn_id):
+    """
+    Initializes synapseclient session and retrieves file metadata
+    """
+    syn = synapseclient.Synapse()
+    syn.login(authToken=os.environ["SYNAPSE_AUTH_TOKEN"])
+    syn_file = syn.get(syn_id)
+    return syn_file
+
+def json_dump(syn_id, test, data):
+    """
+    Dumps dictionary into json file
+    """
+    with open(r'{0}-{1}.json'.format(syn_id, test), "w") as outfile:
```

Comment:
Just in case you're unaware of [Python f-strings](https://peps.python.org/pep-0498/), which were implemented in Python 3.6. I think they're a little more readable and elegant. 

```suggestion
    with open(f'{syn_id}-{test}.json', "w") as outfile:
```